Tools/GNUMake/comps/dpcpp.mak

```
@@ -57,11 +57,7 @@ endif
 
 ########################################################################
 
-ifdef CXXSTD
-  CXXFLAGS += -std=$(strip $(CXXSTD))
-endif
-
-CXXFLAGS += -Wno-error=sycl-strict -fsycl
+CXXFLAGS += -std=c++17 -Wno-error=sycl-strict -fsycl
```

We could also keep, to keep C++20 and C++2b open:
```suggestion
ifdef CXXSTD
  CXXFLAGS += -std=$(strip $(CXXSTD))
else
  CXXFLAGS += -std=c++17
endif

CXXFLAGS += -Wno-error=sycl-strict -fsycl
```

And we just add an `if ($(USE_DPCPP,TRUE))` in WarpX around its setting for `CXXSTD`.

---

Sure, we could do that.